bindings/pyroot/cppyy/CPyCppyy/src/CallContext.h

```diff
@@ -96,6 +97,7 @@ struct CallContext {
 // info/status
     uint64_t fFlags;
     Cppyy::TCppScope_t fCurScope;
+    PyObject* fPyContext;
```

Comment:
I am thinking it would be nice to have somewhere a few more docs about why this is needed. Maybe referincing the linked issue and saying the context will be used to set a  lifeline to help in keeping the temporary arrays alive? 
 Maybe, Here could be a good place?

Reply:
Ok I'll add some explanation!